packages/admin-stories/src/docs/form/Overview.stories.mdx

```diff
@@ -1,15 +1,295 @@
+                            name: values.name
+                        }
+                    }
+                })
+            }}
+        >
+            <Field label="Name" name="name" component={FinalFormInput} />
+            <SaveButton type="submit" />
+        </FinalForm>
+`}
+/>
+
+#### Mutation Variables
+
+A mutation should have an `input` variable that bundles all changeable attributes.
+Attributes that can't be changed should be solitary variables (e.g., `id` or other variables used for identifying a specific resource).
+
+<Source
+    language="tsx"
+    code={dedent`
+        const updateEmployeeMutation = gql\`
+            mutation UpdateEmployee($id: ID!, $input: UpdateEmployeeInput!) {
+                updateEmployee(id: $id, input: $input) {
+                    id
+                    firstname
+                    lastname
+                }
+            }
+        \`;
+        // ...
+        const apolloClient = useApolloClient();
+        // ...
+        <FinalForm
+            mode="add"
+            onSubmit={async (values) => {
+                await apolloClient.mutate({
+                    mutation: updateEmployeeMutation,
+                    variables: {
+                        id: employeeId,
+                        input: {
+                            firstname: values.firstname,
+                            lastname: values.lastname
+                        }
+                    }
+                })
+            }}
+        >
+            <Field label="Firstname" name="firstname" component={FinalFormInput} />
+            <Field label="Lastname" name="lastname" component={FinalFormInput} />
+            <SaveButton type="submit" />
+        </FinalForm>
+`}
+/>
+
+#### Form Submission Example
+
+**Note:** You have to enter "John" and "Doe" for this example to work.
+
+<Canvas>
+    <Story id="stories-form-finalform--submit-mutation-best-practices" />
+</Canvas>
+
+### Context-based Features
+
+Comet Admin `FinalForm` uses React Contexts to interact with other Comet Admin components.
+
+#### FinalForm in a Table
+
+If your `FinalForm` is located inside a [TableQuery](/story/docs-components-table-tablequery--page) component, the tableQuery is refetched automatically after submission.
+
+#### FinalForm in a Stack
+
+If a `FinalForm` is used inside a [Stack](/story/docs-components-stack--page), it automatically navigates back to the previous `StackPage`.
+
+<Canvas>
+    <Story id="stories-form-context-based-features--finalform-in-stack" />
+</Canvas>
+
+This is not always desired.
+For example, if the FinalForm isn't located on an own StackPage.
+In this case you have to override the `onAfterSubmit()` function of `FinalForm`.
+
+<Source
+    language="tsx"
+    code={dedent`
+    <FinalForm
+        // ...
+        onAfterSubmit={() => {
+            // noop
+        }}
+    >
+        {/* ... */}
+    </FinalForm>
+`}
+/>
+
+### Differences between Comet Admin `FinalForm` and React Final Form's `Form`
```

Comment:
That section is a good candidate for Overview

Reply:
Since the sections 

- Submit Mutation Best Practices
- Context-based Features
- Differences between Comet Admin `FinalForm` and React Final Form's `Form`

are all directly related to the `FinalForm` component, it doesn't seem quite right to me to split them up. 
Then we would have the information on the `FinalForm` component scattered over two pages. 

I now moved all three sections to a separate `FinalForm` Component page and only used the overview to provide some info and link to the pages (more like a table of contents)